scenarios/csharp/dotnetcore/call-center/call-center/RestHelper.cs

```diff
@@ -0,0 +1,87 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+//
+
+using System;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CallCenter
+{
+    public static class RestHelper
+    {
+        public static async Task<(HttpResponseMessage, string)> SendGet(string uri, string key, HttpStatusCode[] expectedStatusCodes)
```

Comment:
>HttpResponseMessage [](http://example.com/codeflow?start=34&length=19)

C# 7+ should allow you to "name" Tuple constituents, e.g. `Task<(HttpResponseMessage ResponseMessage, string OtherThing)>` -- which would then let you consume `.ResponseMessage` instead of `.Item1` and the like. Super helpful with non-obvious tuples, as is here--I'm actually still not sure what `response_2` (and thus `.Item2`) actually is!

Reply:
Thanks, that feels more like F# records which is nice. I did this and got rid of all the .Item1/.Item2s.